source/blueprints/user/routes.py

```diff
@@ -121,3 +122,31 @@ def change_password():
         },
     ).json()
 
+
+@user_bp.route("/delete_account", methods=["DELETE"]) 
+@jwt_required()
+def user_del():
+    """Allow the user to delete his own account"""
+
+    data = request.json
+
+    try:
+        delete_confirm = UserDeleteDTO().load(data)
+        delete_user(delete_confirm["confirm_deletion"])
+        db.session.commit()
+
+    except ValueError as error:
+        db.session.rollback()
+        abort(422, {"type": CauseTypeError.VALIDATION_ERROR.value, "data": str(error)})
+        
+    return DataResponse(
+        "User deletion succesfully",
+        200,
+        {
```

Comment:
Just for a matter of keeping the patterns, in the data response dict replace the keys "User", "Profile", "Game Stats" with "user", "profile" and "game_stats", respectively.